mpmp/test_config.py

```diff
@@ -0,0 +1,22 @@
+"""
+Configuration information for tests
+
+"""
+import pathlib
+
+repo_root = pathlib.Path(__file__).resolve().parent.parent
+
+# location of data relevant to running tests
+test_data_dir = repo_root / 'tests' / 'data'
+test_pancan_data = test_data_dir / 'pancancer_data_subsampled.pkl'
```

Comment:
Just double-checking, is the `pancancer_data_subsampled.pkl` file that you uploaded supposed to be empty?

Reply:
It shouldn't be! When I look at the file [here](https://github.com/greenelab/mpmp/blob/f9cc862dd83952b5d33b51baca063021253bb4d2/tests/data/pancancer_data_subsampled.pkl) it looks like it's a ~2 MB binary file - are you seeing something different somewhere?

Reply:
All set, this file should be correct (as far as I can tell)